src/Modal.js

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 const React = require('react');
 const ReactDOM = require('react-dom');
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
```

Comment:
Unused import?

Reply:
ah, good spot. There were only a couple of linting errors in that file so I have just gone through & delinted it & removed it from the ignore list.

Reply:
Wonderful, thanks for doing that! 😁